huxley/core/models.py

```diff
@@ -150,11 +150,7 @@ def update_country_preferences(self, country_ids):
     @classmethod
     def update_fees(cls, **kwargs):
         school = kwargs['instance']
-        delegate_fees = cls.DELEGATE_FEE * sum((
-            school.beginner_delegates,
-            school.intermediate_delegates,
-            school.advanced_delegates,
-        ))
+        delegate_fees = cls.DELEGATE_FEE * (len(Assignment.objects.filter(school_id=school.id, is_double=False))+2*len(Assignment.objects.filter(school_id=school.id, is_double=True)))
```

Comment:
Since these fees are based off number of assignments, not number of delegates, we should rename `DELEGATE_FEE` to be `ASSIGNMENT_FEE`